net/socket.h

```diff
@@ -37,7 +37,6 @@ LogBuffer& operator << (LogBuffer& log, const in_addr& iaddr);
 LogBuffer& operator << (LogBuffer& log, const sockaddr_in& addr);
 LogBuffer& operator << (LogBuffer& log, const in6_addr& iaddr);
 LogBuffer& operator << (LogBuffer& log, const sockaddr_in6& addr);
-LogBuffer& operator << (LogBuffer& log, const sockaddr& addr);
```

Comment:
In most case, `sockaddr` is converted from `sockaddr_storage`, and is always suppose to be used as a pointer.
Print its object will lose information.